cmd/forwarder/config.go

```diff
@@ -66,3 +69,9 @@ func NewIssConfig() (IssConfig, error) {
 
 	return config, nil
 }
+
+func (c IssConfig) LogAuthUser(u string) bool {
+	return c.ValidTokenUser != "" &&
+		u != c.ValidTokenUser &&
+		rand.Intn(99)+1 <= c.TokenUserSamplePct
```

Comment:
I'd suggest using a ratio from zero to one instead of a "percent". Percentages are weird and arbitrary.

eg:

```go
TokenUserSampleRatio float `env:"TOKEN_USER_SAMPLE_RATIO,default=0`
...
rand.Float <= c.TokenUserSampleRatio
```

.. Then the magic number "99" just goes away.